ui/src/features/Editor/components/OverlayUI/components/DebugPreview/index.tsx

```
@@ -0,0 +1,112 @@
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from "@flow/components";
+import { useT } from "@flow/lib/i18n";
+
+import { DataTable, Map } from "./components";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)